toolkit/tools/boilerplate/boilerplate.go

```diff
@@ -20,13 +21,17 @@ var (
 
 	logFile  = exe.LogFileFlag(app)
 	logLevel = exe.LogLevelFlag(app)
+
+	timestampFile = app.Flag("timestamp-file", "File that stores timestamps for this program.").Required().String() // Need to add "--timestamp-file=boilerplate.csv" when try to run boilerplate.go.
```

Comment:
> // Need to add "--timestamp-file=boilerplate.csv" when try to run boilerplate.go. [](http://example.com/codeflow?start=112&length=82)

NIt-pick: no need to mention that - the tool should print info that this argument is required anyway.